coala_quickstart/generation/FileGlobs.py

```diff
@@ -24,7 +24,12 @@ def get_project_files(log_printer, printer, project_dir, non_interactive=False):
     file_globs = ["**"]
 
     ignore_globs = None
-    if os.path.isfile(os.path.join(project_dir, ".gitignore")):
+    flag_gitignore = 0
+    for dirName, subdirName, fileList in os.walk(project_dir):
+        if os.path.isfile(os.path.join(dirName, ".gitignore")):
+            flag_gitignore = 1
```

Comment:
@satwikkansal .gitignore IS relative to the subdirectory. Confirmed it by trying.